src/python/py/models/README.md

```diff
@@ -100,6 +100,20 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p int4 -e execution_provider -c cache_dir_to_store_temp_files
 ```
 
+### LoRA Models
```

Comment:
Since this uses `extra_options`, can you move this section to be after "Use QDQ Pattern for Quantization" and add a reference to this section in the table of contents after "Use QDQ Pattern for Quantization"?

https://github.com/microsoft/onnxruntime-genai/blob/dfbe14c39bc0486e1289332bca2003ff66a74fc7/src/python/py/models/README.md?plain=1#L5-L26

Can you also change the title's size so that it matches the title sizes that are sub-sections in the "Extra Options"?

```suggestion
#### LoRA Models
```